jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/api/EnricherContext.java

```diff
@@ -135,4 +136,6 @@ default Optional<String> getDependencyVersion(String groupId, String artifactId)
      * @return the Handler Hub instance.
      */
     HandlerHub getHandlerHub();
+
+    JKubeBuildStrategy getBuildStrategy();
```

Comment:
Maybe this field could be declared on the [`Configuration`](https://github.com/eclipse/jkube/blob/682cc6f25ccf3e95e20832d1396fa3890f3a4e26/jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/api/model/Configuration.java)

WDYT?

Reply:
I added it to Configuration, but I couldn't remove it from JKubeEnricherContext as it's getting initialized there. For now I've removed it from EnricherContext interface

https://github.com/eclipse/jkube/blob/682cc6f25ccf3e95e20832d1396fa3890f3a4e26/jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/api/JKubeEnricherContext.java#L75